config/routes.rb

```diff
@@ -106,6 +106,7 @@ def self.matches?(request)
     delete "linked_users/:id", to: "linked_users#destroy", as: "destroy_linked_user"
 
     resources :reports, only: [:create, :new, :destroy, :update]
+    get "reports/:id/:status", to: "reports#update", as: "report_submit"
```

Comment:
This creates the possibility for a CSRF attack, but the potential consequences are minor.